xarray/core/dataarray.py

```diff
@@ -1214,6 +1214,56 @@ def fillna(self, value):
         out = ops.fillna(self, value)
         return out
 
+    def interpolate_na(self, dim=None, method='linear', limit=None,
+                       use_coordinate=True,
+                       **kwargs):
+        """Interpolate values according to different methods.
+
+        Parameters
+        ----------
+        dim : str
+            Specifies the dimension along which to interpolate.
+        method : {'linear', 'time', 'index', 'values', 'nearest'}
+            'linear': ignore the index and treat the values as equally
+                      spaced. default
```

Comment:
This is a minor point, but can we format this a little more consistently?

The `compat` arg on the Dataset constructor gets formatted nicely into a list by Sphinx and is still pretty readable as plain text:
https://github.com/pydata/xarray/blob/0818a3670521e7dc06b75b0a5241d2cf3f522cb2/xarray/core/dataset.py#L342-L350